modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-querying-the-policy-compliance-status-for-each-cluster_{context}"]
+= Querying the policy compliance status for each cluster
+
+After you have created the validator inform policies for your clusters and pushed them to the
+the zero touch provisioning (ZTP) Git repository, you can check the status of each cluster for policy compliance.
+
+.Procedure
+
+. To query the status of the spoke clusters using either the {rh-rhacm-first} web console or the CLI:
++
+* To query status from the {rh-rhacm} web console, click *Governance => Find policies =>
+Search “du-validator-policy” => Click into the policy*.
+
+* To query status using the CLI, run the following command:
++
+[source,terminal]
+----
+$ oc get policies du-validator-policy -n <namespace-for-common> -o jsonpath={'.status.status'} | jq
```

Comment:
[Underscores for replaceables](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output): `<namespace_for_common>`

Reply:
changed